README.md

```diff
@@ -219,6 +219,26 @@ Set `trackEcommerce: false` for an account object that is not tracking e-commerc
   // RegEx to scrub location before sending to analytics.
   // Internally replaces all matching segments with an empty string.
   AnalyticsProvider.setRemoveRegExp(/\/\d+?$/);
+  
+    // Activate reading custom tracking urls from $routeProvider config (default is false)
```

Comment:
Man, you are picky. ;)

I swear after last time I looked over the code three times to avoid this situation. :-)